test/gpu/jit.cpp

```diff
@@ -206,8 +206,10 @@ TEST_CASE(compile_warnings)
     EXPECT(not compile("").empty());
     EXPECT(not compile("-Wunused-parameter -Wno-error").empty());
     EXPECT(not compile("-Wno-unused-parameter -Werror").empty());
+#ifndef MIGRAPHX_USE_HIPRTC
     EXPECT(test::throws([&] { compile("-Werror=unused-parameter"); }));
     EXPECT(test::throws([&] { compile("-Wunused-parameter -Werror"); }));
+#endif
```

Comment:
We should only disable these tests when the workarounds are enabled.

Reply:
Done.